docs/Pflichtenheft.tex

```diff
@@ -65,22 +65,26 @@ \section{Zielbestimmung}
 \subsection{Musskriterien}
 \subsubsection{Allgemein}
 \begin{itemize}
-  \item Die gesamte Kommunikation zwischen \gls{Uberwachungskonsole} und \gls{Fertigungssimulation} findet \"uber das \gls{OPC UA} Protokoll statt.
-  \item \gls{Fertigungssimulation} und \gls{Uberwachungskonsole} laufen auf getrennten Rechnern.
+  \item Die Kommunikation zwischen \gls{Uberwachungskonsole} und \gls{Fertigungssimulation} findet \"uber \glspl{OPC UA Server} auf Seiten der \gls{Fertigungssimulation}
+    und \glspl{OPC UA Client} auf Seiten der \gls{Uberwachungskonsole} statt.
+  \item Die \gls{Fertigungssimulation} mit den \glspl{OPC UA Server} und die \gls{Uberwachungskonsole} mit den \glspl{OPC UA Client} laufen auf getrennten Rechnern.
 \end{itemize}
 
 \subsubsection{Überwachungskonsole}
 \begin{itemize}
   \item Die \gls{Uberwachungskonsole} ruft alle \glspl{Sensordatum} der \gls{Fertigungssimulation} in einem festen Zeitintervall über einen \gls{OPC UA} Client ab.
   \item Der \gls{OPC UA} Client ist mit \gls{OPC UA} Servern verbunden. Dazu ist die Einstellung der Verbindungsinformationen möglich.
-  \item Die abgerufenen Werte und deren zeitlicher Verlauf werden graphisch dargestellt. Die Werte sind Zu- und Abflussmengen, Temperaturen, Flüssigkeitsfarben sowie die Drehzahl eines Mischermotors.
+  \item Die von den \glspl{OPC UA Client} abgerufenen Werte und deren zeitlicher Verlauf werden graphisch dargestellt. Die Werte sind Zu- und Abflussmengen, Temperaturen,
+    Flüssigkeitsfarben sowie die Drehzahl eines Mischermotors.
   \item Die \gls{Uberwachungskonsole} enthält Alarme, die beim Überschreiten von Schwellenwerten in der \gls{Fertigungssimulation} durch den \gls{OPC UA} Server gesendet
     und dem Nutzer graphisch mitgeteilt werden.
 \end{itemize}
 
 \subsubsection{Fertigungssimulation}
 \begin{itemize}
-  \item In der \gls{Fertigungssimulation} kann der Benutzer die simulierte Fertigung steuern.
+  \item Die \gls{Fertigungssimulation} enth\"alt eine schematische Darstellung der simulierten \gls{Produktionsanlage}.
+  \item In der \gls{GUI} der \gls{Fertigungssimulation} kann der Benutzer die simulierte \gls{Produktionsanlage} steuern.
+  \item Die \gls{Fertigungssimulation} setzt die \glspl{Sensordatum} in den \glspl{OPC UA Server}.
```

Comment:
"setzt" klingt komisch, finde ich. Und es muss heißen "in DIE \glspl{OPC UA Server}."